Kinetic_space_partition/examples/Kinetic_space_partition/CMakeLists.txt

```
@@ -0,0 +1,39 @@
+# Created by the script cgal_create_CMakeLists.
+# This is the CMake script for compiling a set of CGAL applications.
+
+cmake_minimum_required(VERSION 3.1...3.15)
```

Use the same as in CGAL `master`:

```suggestion
cmake_minimum_required(VERSION 3.1...3.23)
```

otherwise there is an issue with CMake 3.19.